fflib/src/classes/fflib_QueryFactoryTest.cls

```diff
@@ -151,6 +151,45 @@ private class fflib_QueryFactoryTest {
 		System.assert( Pattern.matches('SELECT.*Email.*FROM.*',query), 'Expected Name field in query, got '+query);
 	}
 
+	@isTest
+	static void orderingSet(){
```

Comment:
**OPTIONAL: Rename test method**
It would be nice if the test method name was more reflective of the method under test and the expected behaviour.
I suggest `static void setOrdering_ReplacesPreviousOrderingsWithExpectedOrdering()`